deploy-agent/deployd/agent.py

```diff
@@ -51,6 +53,51 @@ class AgentRunMode(object):
     def is_serverless(mode):
         return AgentRunMode.SERVERLESS == mode
 
+class TimeElapsed():
```

Comment:
this is a good candidate for some unit tests

Reply:
and a docstring

Reply:
updated per req, thank you.